src/Prestige.Identity.WebApi/Controllers/IdentityController.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Prestige.Identity.Common.Commands;
+using Prestige.Identity.Infrastructure.Services;
+using System.Threading.Tasks;
+
+namespace Prestige.Identity.WebApi.Controllers
+{
+    [Route("identity")]
+    [ApiController]
```

Comment:
We need to extract both attributes to the CommonController because we want each of our controllers to inherit the `[ApiController]` attribute